### DISCUSSION TITLE
How can we better leverage our project board?

### DISCUSSION THREAD

**Discussion by asteiker** [earthaccess-dev/earthaccess#894] [General]
I experimented with an earthaccess GitHub project quite a while back, but I'm not aware of any active use: https://github.com/orgs/nsidc/projects/10/views/1. I think this could be a great way to provide transparency into our work status and prioritization of bugs and new features. Thoughts on utilizing this board? How can I help make this more useful to our community?

**Comment by mfisher87** [comment]
I think conceptually the idea is solid -- visualizing the work in a richer way, reflecting the state of the work in the visualization. Kanban!

Perhaps we need more purpose-built boards? I'm thinking about one for items that need to be solved by a decision committee (speaking of which, let's get a meeting on the calendar -- I think now is the time for an official project calendar since we have >1 thing, and [we can use Google Calendar in the way the Jupyter project does](https://discourse.jupyter.org/t/jupyter-community-calendar/2485)).

Perhaps a hack day board? That seems less useful, but maybe a good way to reinforce engagement in GitHub?

I think these kinds of things just need to be boldly tried to see if they work. And a failure doesn't necessarily mean the idea is bad, maybe we just didn't communicate right, or didn't communicate enough? So maybe all we need are some small tweaks to make it work.

I think we should find *repeating* touch points to look at the board together, and from there we can create a feedback loop about what's working and what's not working.

**Comment by asteiker** [comment]
I'm starting to experiment with a Decision committee board but quickly running into a GH limitation. Unfortunately I cannot add Discussions to projects (see https://github.com/orgs/community/discussions/11223 and https://github.com/orgs/community/discussions/3775). This could limit our ability to surface both discussions and issues needing the attention of the decision committee unless we convert existing discussions to issues.

**Reply by mfisher87** [reply]
Maybe decision committee involvement is a sign we're serious about working a feature and the discussion is ready to be converted to an issue?

**Reply by asteiker** [reply]
💯 Great idea!
